inc/notificationtargetobjectlock.class.php

```diff
@@ -110,8 +111,9 @@ function addDataForTemplate($event, $options = []) {
       $this->data['##objectlock.name##']     = $object->fields['name'];
       $this->data['##objectlock.id##']       = $options['item']->fields['items_id'];
       $this->data['##objectlock.type##']     = $options['item']->fields['itemtype'];
-      $this->data['##objectlock.date_mod##'] = Html::convDateTime($options['item']->fields['date_mod'],
+      $this->data['##objectlock.date##']     = Html::convDateTime($options['item']->fields['date_mod'],
                                                                    $user->fields['date_format']);
+      $this->data['##objectlock.date_mod##'] = $this->data['##objectlock.date##'];
```

Comment:
Should not you use `Html::conDateTime`?
Also, `##objectlock.date##` on `date_mod` and `##objectlock.date_mod##` on date sounds a bit weird